spring-kafka-test/src/main/java/org/springframework/kafka/test/rule/KafkaEmbedded.java

```diff
@@ -457,7 +475,7 @@ public boolean isEmbedded() {
 	 * @throws Exception an exception.
 	 */
 	public void consumeFromAllEmbeddedTopics(Consumer<?, ?> consumer) throws Exception {
-		consumeFromEmbeddedTopics(consumer, this.topics);
+		consumeFromEmbeddedTopics(consumer, this.topics.toArray(new String[this.topics.size()]));
```

Comment:
My latest IDEA doesn't recommend me to use arrays with static size for this operation:
![image](https://user-images.githubusercontent.com/1071590/40121284-3292bd32-58ef-11e8-903c-d8706ec3f2c5.png)

Reply:
Thank you, updated my intellij and saw the same warning. Should be better now.